<?xml version="1.0"?>
<launch>
  <arg name="camera_frame_id" default="camera" /> 
  <arg name="lidar_frame_id" default="lidar_front_base_link" /> 
  <arg name="camera_info_topic" default="/camera_info" />

  <node pkg="manual_lidar_camera_calibration" name="extrinsic_calibrator_node" exec="extrinsic_calibrator.py" output="screen">
    <param name="camera_frame_id" value="$(var camera_frame_id)" />
    <param name="lidar_frame_id" value="$(var lidar_frame_id)" />
    <param name="camera_info_topic" value="$(var camera_info_topic)" />
  </node>

</launch>
